tests/test_langscripts.py

```diff
@@ -0,0 +1,57 @@
+from __future__ import annotations
+
+import pytest
+from attr import dataclass
+
+from mteb.abstasks.languages import LanguageScripts
+
+
+@dataclass
+class LangScriptTestCase:
+    args: dict
+    contains_language: list[str]
+    not_contains_language: list[str]
+    contains_script: list[str]
+    not_contains_script: list[str]
+
+
+test_cases = [
+    LangScriptTestCase(
+        args=dict(languages=["fra"], scripts=None),
+        contains_language=["fra", "fra-Latn"],
+        not_contains_language=["eng"],
+        contains_script=[],
+        not_contains_script=["Latn"],
+    ),
```

Comment:
I didn't find this intuitive - maybe I am not as familiar with how the languages are defined. If `contains_language=["fra", "fra-Latn"],`, why wouldn't `contains_script=["Latn"]` be true?

Reply:
dict(languages=["fra"], scripts=None)
does not contain latn.

It would be impossible to know without having a list of languages and corresponding scripts.

Reply:
I see, thanks.